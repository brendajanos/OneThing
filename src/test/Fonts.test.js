import FontPicker from 'font-picker-react';
import renderer from 'react-test-renderer';

describe("Setting the pomodoro timer component", () => {
  it("matches the snapshot", () =>{
    const tree = renderer.create(<FontPicker />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})