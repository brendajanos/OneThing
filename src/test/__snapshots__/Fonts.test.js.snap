// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Setting the pomodoro timer component matches the snapshot 1`] = `
<div
  className=""
  id="font-picker"
>
  <button
    className="dropdown-button"
    onClick={[Function]}
    onKeyPress={[Function]}
    type="button"
  >
    <p
      className="dropdown-font-family"
    >
      Open Sans
    </p>
    <p
      className="dropdown-icon loading"
    />
  </button>
</div>
`;
